wordpress-mobile/WordPress-Android
# Issue/11924 remove whats next feature flag

Fixes #11924

A couple of things in this PR:
- Removed the feature flag.
- Removed test announcement.
- An empty json file is now OK (was booked by validation test before and crashing App Settings if ignored.)
- Changed example json to be less crazy.

To test:
- Confirm that the feature works as expected with vanilla build (note that vanilla build has a separate versionCode).

PR submission checklist:
